src/request.c

```diff
@@ -177,26 +170,27 @@ void sendGroupSpecificMemberQuery(void *argument) {
     }
 
     /**
-     * FIXME: This loops through all interfaces the group is active on an sends queries.
-     *        It might be better to send only a query on the interface the leave was accepted on and remove only that interface from the route.
+     * FIXME:
+     * This loops through all interfaces the group is active on and sends
+     * queries.
+     *
+     * It might be better to send only a query on the interface the leave
+     * was accepted on and remove only that interface from the route.
```

Comment:
This is not good reformat. Second sentence is also part of FIXME, but after change is not visually anymore.